src/main/reseal.c

```diff
@@ -407,31 +404,38 @@ int main(int argc, char **argv)
     kmyth_clear(ownerAuthPasswd, oa_passwd_len);
     return 1;
   }
-  // We are done with authString and ownerAuthPasswd, so clear them
-  //kmyth_clear(authString, auth_string_len);
-  //kmyth_clear(ownerAuthPasswd, oa_passwd_len);
 
-  if (stdout_flag == true)
+  // rename the input file **** This is temporary. Needs to be changed slightly
+  int worked;
+  char *newFileName = "orginalFile.ski";
```

Comment:
This renames the input to a "fixed" file name. If we run `kmyth-reseal` on multiple .ski input files, we will have to rename this result between every invocation of `kmyth-reseal` or the names will conflict. Note: this is also missing the code to check for that filename collision.

Is there a reason that you think this is better than the proposed solution that renamed the input file to <input filename>.orig?